### PR TITLE
Only displays publication label when publications.

### LINKED ISSUE #203: Don't display Publications label when no publications
screen shot 2018-11-09 at 4 50 15 pm

### PR DESCRIPTION
**Pull request by justinlittman**
closes #203

### CODE REVIEW

**Comment by justinlittman** [comment]
Do I get points for creativity, @jcoyne?

File: app/models/solr_document.rb [hunk 1]
```diff
@@ -72,6 +72,7 @@ def linked_publications(params)
                                                    search_builder_class: PublicationSearchBuilder)
     response, _deprecated_stuff = search_service.search_results
     docs = response.documents
```
Comment (jcoyne) [inline app/models/solr_document.rb]: Should we cache this so it doesn't make 2 of these queries for each page draw?

Comment (justinlittman) [reply app/models/solr_document.rb]: Probably not necessary for the anticipated 1 simultaneous user. But I defer to your judgement and recommendation on how/where to cache.

Comment (jcoyne) [reply app/models/solr_document.rb]: How about something like this?
```ruby
  return @publications unless @publications.nil?
  response, _deprecated_stuff = search_service.search_results
    docs = response.documents
    return @publications = false if docs.empty?
    tuples = docs.map do |doc|
      "<li><a href=\"#{search_link(doc.id)}\">#{doc.title}</a> (#{doc.first('created_year_isim')})</li>"
    end
    @publications = "<ul>#{tuples.join}</ul>".html_safe
end

Comment (justinlittman) [reply app/models/solr_document.rb]: Works for me. Fixing.

Comment (justinlittman) [reply app/models/solr_document.rb]: Required a little bit of tweaking because returning false wasn't equivalent to returning nil.

**Pull request merged**